nettle/station_set.py

```diff
@@ -488,16 +488,18 @@ def _get_old_metadata(self):
 
         return old_station_metadata, old_stations, old_hash
 
-    def station_metadata_to_geojson(self, data, **kwargs):
+    def station_metadata_to_geojson(self, data: dict, md_path:str, **kwargs):
         '''
-        Take the station metadata self.STATION_DICT and convert it to valid geojson
+        Take the station metadata (presumed self.STATION_DICT) and convert it to valid JSON and geojson
         '''
-        old_station_metadata, old_stations, old_hash = self._get_old_metadata()
+        if data == None:
+            data = self.STATION_DICT
 
+        old_station_metadata, old_stations, old_hash = self._get_old_metadata(md_path=md_path)
         geojson = {"type": "FeatureCollection", "features": []}
 
         self.geo_json_handler.append_features(geojson, old_stations, old_hash,
-                                              self.STATION_DICT.items())
+                                              data.items())
```

Comment:
Conceptually, self.STATION_DICT would be the output that will generate stations.json. Not sure if this is going to keep working on the other etls as data is a generic dict to carry specific etl info

Reply:
don't think it will work out of the box for all existing etls. 

first of: if people use def parse from nettle this that you wrote it here,
```
        if data == None:
            data = self.STATION_DICT
```
you will at least have data['stations_ids'] that is set here:
https://github.com/Arbol-Project/nettle/blob/main/nettle/station_set.py#L627
and iterated here : https://github.com/Arbol-Project/nettle/blob/main/nettle/station_set.py#L706

But for the .append_features, for example, we fix the unit of measurement in speedwell for STATION_DICT, but not in data
https://github.com/Arbol-Project/non-gridded-etl-managers/blob/main/non_gridded_etl_managers/speedwell.py#L370

Reply:
In ghcnd we transform data to STATION_DICT, but not itself: 
https://github.com/Arbol-Project/non-gridded-etl-managers/blob/main/non_gridded_etl_managers/ghcnd.py#L1182